velox/connectors/hive/HiveConfig.h

```diff
@@ -103,9 +103,10 @@ class HiveConfig {
   /// The max coalesce bytes for a request.
   static constexpr const char* kMaxCoalescedBytes = "max-coalesced-bytes";
 
-  /// The max coalesce distance bytes for combining requests.
-  static constexpr const char* kMaxCoalescedDistanceBytes =
-      "max-coalesced-distance-bytes";
+  /// The max merge distance combining read requests.
+  static constexpr const char* kMaxCoalescedDistance = "max-coalesced-distance";
+  static constexpr const char* kOrcMaxMergeDistanceSession =
```

Comment:
Can we rename this as well

Reply:
You mean kOrcMaxMergeDistanceSession?
No, we cannot do this - I don't believe we have system in place to convert session properties from Presto to Velox.
And I'm not convinced we should come up with one.